### DISCUSSION TITLE
/livewire/message/{name} not found 404

### DISCUSSION THREAD

**Discussion by ghost** [livewire/livewire#5495] [1. Help] [answered]
hello there,
I'm facing a issue only on live-server "/livewire/message/{name} not found 404" with by default post request ,when I open this URL on my browser livewire show me error `The livewire/message/products method is not supported for route GET. Supported methods: POST.`
but it's working fine in with localhost , I tried every possible solutions

**Comment by joshhanley** [comment, marked as answer]
@moinshaikh4607 there was a bug released in v2.10.8. I have submitted a PR https://github.com/livewire/livewire/pull/5490 with a fix. Hope this helps!

**Reply by ghost** [reply]
Thank you so much @joshhanley , but I've still a confusion this is temporary solutions right because **livewire** have not fixed yet ?

**Reply by joshhanley** [reply]
@moinshaikh4607 temporary solution would be to lock your livewire version to 2.10.7. 

Once I’ve finalised that PR and it’s been merged and a release tagged, then you can remove the version restriction 🙂

**Reply by RealMrHex** [reply]
> @moinshaikh4607 temporary solution would be to lock your livewire version to 2.10.7.
> 
> Once I’ve finalised that PR and it’s been merged and a release tagged, then you can remove the version restriction 🙂

Is this problem fixed?
I still got 419 Error!
![image](https://user-images.githubusercontent.com/62525184/211950714-fbd31f15-a627-4b62-87e0-b9cfde9245f6.png)
![image](https://user-images.githubusercontent.com/62525184/211950811-f007c4b3-7bdf-47c9-9ce5-fc0543ab6516.png)

**Reply by joshhanley** [reply]
@RealMrHex not quite, please see my comment above about version locking. 

But 419 is probably a different issue, this issue is about 404 routes not found.

**Reply by RealMrHex** [reply]
@joshhanley it was my fault, solved now ;)
and also there is no more 404 error with the latest version of laravel sail [latest] and livewire [2.10.8] in the local env.
idk why it fixed but ill update you if i found sth else.
Thanks

**Reply by joshhanley** [reply]
@RealMrHex no worries! 🙂
